pom.xml

```diff
@@ -241,6 +241,7 @@
                         <goals>
                             <goal>check</goal>
                         </goals>
+                        <phase>process-classes</phase>
```

Comment:
Done - I think it is worth doing (checkstyle after tests is a bit of a bummer), but will put that in another PR